docs/migration.md

```diff
@@ -54,17 +54,13 @@ This document offers guidance on migrating from Prometheus 2.x to Prometheus 3.0
 
 - The `.` pattern in regular expressions in PromQL matches newline characters. 
   With this change a regular expressions like `.*` matches strings that include 
-  `\n`. This applies to matchers in queries and relabel configs. For example the 
-  following regular expressions now match the accompanying strings, whereas in 
-  Prometheus v2 these combinations didn't match.
-
-  | Regex      | Additional matches  |
```

Comment:
Is it really more readable without table? The text feels quite dense now:

<img width="991" alt="image" src="https://github.com/user-attachments/assets/4c0d8db1-b062-4dc0-93ae-9c824031e98f">

Reply:
I think the table would be better, but I didn't figure out a way to get it to display properly on the website. I think tables don't work well with nanoc/kramdown when they're indented within bulleted lists.

Currently the PromQL section looks like:

<img width="897" alt="image" src="https://github.com/user-attachments/assets/53e84622-dee6-4911-b263-12c7cca8687e">

With the table section messed up